src/nvim/globals.h

```diff
@@ -1033,6 +1033,9 @@ EXTERN char_u e_unsupportedoption[] INIT(= N_("E519: Option not supported"));
 EXTERN char_u e_fnametoolong[] INIT(= N_("E856: Filename too long"));
 EXTERN char_u e_float_as_string[] INIT(= N_("E806: using Float as a String"));
 
+EXTERN char_u e_noserver[] INIT(=N_(
+    "E247: couldn't connect to remote server: %s"));
```

Comment:
```suggestion
    "E247: failed to connect to remote server: %s"));
```

or `cannot`. Let's avoid contractions in error messages